client/components/credit-card-form-fields/index.jsx

```diff
@@ -212,7 +212,7 @@ export class CreditCardFormFields extends React.Component {
 				<div className={ creditCardFormFieldsExtrasClassNames }>
 					{ this.createField( 'expiration-date', Input, {
 						inputMode: 'numeric',
-						label: translate( 'MM/YY', {
+						label: translate( 'Expiry: MM/YY', {
```

Comment:
:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?

Reply:
This make sense - we can lose the context now that the string itself is more explicit